roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,12 @@
 dependencies:
   - pip
+  - role: logrotate
+    logrotate_configs:
```

Comment:
Putting the dict in the zuul role's defaults doesn't work, but putting it in the zuul group_vars does.